### PR TITLE
Split user and client API

### PR DESCRIPTION
**Pull request by mmadariaga**
Extract user endpoints from client API

#### Type Of Change 
- [ ] Small bug fix
- [X] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX)

### CODE REVIEW

File: web/rest/user/config/api/raw/provider.yml [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+Ivoz\Provider\Domain\Model\Feature\Feature:
+  itemOperations: []
+  collectionOperations: []
+
+Ivoz\Provider\Domain\Model\RatingPlanGroup\RatingPlanGroup:
+  attributes:
+    access_control: 'FALSE'
+    pagination_enabled: false
+  itemOperations: []
+  collectionOperations: []
```
Comment (Kaian) [inline web/rest/user/config/api/raw/provider.yml:100]: This represent a group of Destination Rates for the billing engine, not sure why it's required in User API

File: web/rest/user/config/packages/prod/deprecations.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# As of Symfony 5.1, deprecations are logged in the dedicated "deprecation" channel when it exists
+#monolog:
+#    channels: [deprecation]
+#    handlers:
+#        deprecation:
+#            type: stream
+#            channels: [deprecation]
+#            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
```
Comment (Kaian) [inline web/rest/user/config/packages/prod/deprecations.yaml:8]: More empty files :thinking:

**Review by Kaian — APPROVED** [review]

File: web/rest/user/config/api/raw/provider.yml [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        parameters: []
+  attributes:
+    access_control: '"ROLE_COMPANY_USER" in roles'
+    swagger_context:
+      required:
+      - company
+
+Ivoz\Provider\Domain\Model\Feature\Feature:
+  itemOperations: []
+  collectionOperations: []
```
[the remark this answers is not in the document]
Comment (mmadariaga) [reply web/rest/user/config/api/raw/provider.yml]: removed

File: web/rest/user/config/api/raw/provider.yml (hunk 1 above)
Comment (mmadariaga) [reply web/rest/user/config/api/raw/provider.yml]: removed

**Pull request merged**